### ISSUE TITLE
not showing ok

### ISSUE THREAD

**Issue by tescaso** [seanlowe/obsidian-timelines#103]
I don't know why all time I try to create a timeline it looks not ok.

![image](https://github.com/user-attachments/assets/f3aa7a54-e537-4489-a734-e9e5450e0a13)

**Comment by seanlowe** [comment]
Indeed, that looks pretty jank. Could you do a couple things for me:

- go into the plugin settings, scroll to the bottom and turn on debug mode.
  - hit Ctrl-Shift-I and see if there are any warning or error messages? If so, please copy them into a comment.
- could you copy your event here just to double check that side as well?


And just out of curiosity, how big is your Obsidian window? Can't imagine it's that but you never know

**Comment by tescaso** [comment]
I don't see any error in the console. 
I tried with html events and yaml.
This is the html:
<div class="ob-timelines"
  data-title="Our First Event"
  data-start-date="2024-5-24"
>
  Event Number One
</div>
I am using Obsidian under linux

**Comment by tescaso** [comment]
My screen resolution is 1920x1080

**Comment by seanlowe** [comment]
I cannot reproduce this issue - are you using any themes? I've run into issues with themes in the past.

**Comment by sootheyin** [comment]
I also encountered a similar bug. After troubleshooting, I found it was because I was using the Timeline plugin at the same time as this plugin. Disabling the Timeline plugin resolved the issue.
